plugins/mcreator-localization/help/default/particle/max_age.md

```diff
@@ -1 +1 @@
-If particle can expire and is older than this number of, it will expire.
\ No newline at end of file
+The particle will expire as soon as its age is equal to this number of ticks.
```

Comment:
Add a note about age diff here as this statement sounds like age diff has no effect on this parameter.

Reply:
:heavy_check_mark: Fixed